docs/deployment/setup.md

```diff
@@ -0,0 +1,176 @@
+# KVCache Manager Setup Guide
+
+This guide provides instructions for deploying the llm-d-kv-cache-manager with vLLM, LMCache, and Redis using Helm.
```

Comment:
I think this opener should better introduce the scope and agenda of this doc. It should describe what is going to be ran in short, including the example and its content.